addons/calendar/models/mail_activity.py

```diff
@@ -36,7 +36,7 @@ def _action_done(self, feedback=False, attachment_ids=False):
                 description = event.description
                 description = '%s<br />%s' % (
                     description if not tools.is_html_empty(description) else '',
-                    _('Feedback: %(feedback)s', tools.plaintext2html(feedback)) if feedback else '',
+                    _('Feedback: %(feedback)s', feedback=tools.plaintext2html(feedback) if feedback else ''),
```

Comment:
@auon-odoo 
Should we really be changing the behaviour when `feedback` is falsy? (you probably just moved the closing parenthesis while testing your previous version 54778b3a7623fa81a39b1186887465782cd92690)
before: empty string
now : 'Feedback: '

Other than that, looks good, but I don't have the r+ rights unfortunately

Reply:
You're right, thanks @mao-odoo